libpod/image.go

```diff
@@ -41,6 +54,9 @@ type CopyOptions struct {
 	// strip or add signatures to the image when pushing (uploading) the
 	// image to a registry.
 	common.SigningOptions
+
+	// SigningPolicyPath this points to a alternative signature policy file, used mainly for testing
+	SignaturePolicyPath string
```

Comment:
We removed this from PullImage, so this can probably go away?

Reply:
we need it for push right? The signaturePolicyPath in the runtime config is the default, or can that be overwritten by the user?

Reply:
I'm wondering if we want to allow people to override the default for Pull as well as Push. The runtime config policy path is the default, but can be overridden by the user, so I think that should be enough.

Reply:
So if we need it for push, it can stay.

Reply:
Yes we have a hidden option for this right now to allow for testing.  But I think we should handle it for both push and pull.

Reply:
Alright. @umohnani8 Can you add a parameter to override signature policy to pull then? Just pass in the path as a string. if the one the user gives us is empty, use the default one in the libpod runtime.

Reply:
adding

Reply:
fixed.